bundles/core/src/main/java/com/adobe/cq/commerce/core/components/client/MagentoGraphqlClient.java

```diff
@@ -29,7 +29,9 @@
 import com.adobe.cq.commerce.graphql.client.RequestOptions;
```

Comment:
Changes to this file are made so that introspection queries can be made. If memory serves originally I had created a separate `MagentoIntrospectionGraphqlClient` or something similar but the team thought it would be better to just have the existing class handle both queries types (with the default behavior remaining in tact)